apps/web/pages/booking/[uid].tsx

```diff
@@ -287,6 +287,11 @@ export default function Success(props: SuccessProps) {
     bookingInfo.status
   );
 
+  const hasSMSAttendeeAction =
```

Comment:
Why do we need that? Wouldn't it be enough to only check if the bookingInfo has a smsReminderNumber?